src/ui/components/shared/MaterialIcon.tsx

```diff
@@ -18,6 +20,7 @@ function MaterialIcon({
   highlighted,
   className,
   outlined,
+  color,
```

Comment:
As a follow up, we should refactor this component to be less opinionated about color so it can just inherit the parent's.

Reply:
#3538.